astro/src/pages/team/previous.astro

```diff
@@ -0,0 +1,102 @@
+---
+import Layout from '../../layouts/Layout.astro';
+import Section from '../../components/Section.astro';
+---
+
+<Layout title="Our previous contributors">
+ <Section theme="body-tertiary">
+  <div class="col-xxl-9 pb-3">
+   <p class="lead">We sincerely appreciate the time and energy that each person listed below has contributed to our mission.</p>
+  </div>
+  <div>
+   <div>
+    <div class="card mb-3 shadow team-card">
```

Comment:
Let's increase the margin between the cards a bit more so that it matches with the rest of the site and has more vertical space. Perhaps an `mb-5` ?